<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MexTK GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabFighter">
       <attribute name="title">
        <string>Fighter</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Path to Code(s)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>txedit_CodePath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_CodePath_Fighter"/>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectCodeFile_Fighter">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="chkbox_FunctionEnable_Fighter">
            <property name="text">
             <string>Create .dat for ftFunction Node</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>ftFunction .dat file to Create</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>txedit_FunctionPath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_FunctionPath_Fighter">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectFunctionFile_Fighter">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_9">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="chkbox_DatEnable_Fighter">
            <property name="text">
             <string>Add to Fighter .dat file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Path to .dat file</string>
              </property>
              <property name="buddy">
               <cstring>txedit_DatPath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_DatPath_Fighter">
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectDatFile_Fighter">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Compile_Fighter">
              <property name="text">
               <string>Compile Fighter</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabItem">
       <attribute name="title">
        <string>Item</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Path to Code(s)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>txedit_CodePath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_CodePath_Item"/>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectCodeFile_Item">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Item ID</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>txedit_CodePath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_ItemID"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="chkbox_FunctionEnable_Item">
            <property name="text">
             <string>Create .dat for itFunction Node</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>itFunction .dat file to Create</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>txedit_CodePath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_FunctionPath_Item">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectFunctionFile_Item">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="chkbox_DatEnable_Item">
            <property name="text">
             <string>Add to Fighter .dat or Stage .dat file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Path to .dat file</string>
              </property>
              <property name="buddy">
               <cstring>txedit_DatPath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_DatPath_Item"/>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectDatFile_Item">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Compile_Item">
              <property name="text">
               <string>Compile Item</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStage">
       <attribute name="title">
        <string>Stage</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Path to Code(s)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>txedit_CodePath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_CodePath_Stage"/>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectCodeFile_Stage">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="chkbox_FunctionEnable_Stage">
            <property name="text">
             <string>Create .dat for grFunction Node</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>grFunction .dat file to Create</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="buddy">
               <cstring>txedit_CodePath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_FunctionPath_Stage">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectFunctionFile_Stage">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_11">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="chkbox_DatEnable_Stage">
            <property name="text">
             <string>Add to Stage .dat file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Path to .dat file</string>
              </property>
              <property name="buddy">
               <cstring>txedit_DatPath_Fighter</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txedit_DatPath_Stage"/>
            </item>
            <item>
             <widget class="QToolButton" name="btn_SelectDatFile_Stage">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_7">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_Compile_Stage">
              <property name="text">
               <string>Compile Stage</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select kind of code to be compiled</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn_DolphinTest">
      <property name="text">
       <string>Test in Dolphin</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_model"/>
    <addaction name="actionSave_filled_content_to_model"/>
    <addaction name="actionDolphin_settings"/>
    <addaction name="actionMexTK_settings"/>
   </widget>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_filled_content_to_model">
   <property name="text">
    <string>Save filled content to model</string>
   </property>
  </action>
  <action name="actionOpen_model">
   <property name="text">
    <string>Open model...</string>
   </property>
  </action>
  <action name="actionDolphin_settings">
   <property name="text">
    <string>Dolphin settings</string>
   </property>
  </action>
  <action name="actionRecent_1">
   <property name="text">
    <string>Recent 1</string>
   </property>
  </action>
  <action name="actionRecent_2">
   <property name="text">
    <string>Recent 2</string>
   </property>
  </action>
  <action name="actionRecent_3">
   <property name="text">
    <string>Recent 3</string>
   </property>
  </action>
  <action name="actionRecent_4">
   <property name="text">
    <string>Recent 4</string>
   </property>
  </action>
  <action name="actionRecent_5">
   <property name="text">
    <string>Recent 5</string>
   </property>
  </action>
  <action name="actionMexTK_settings">
   <property name="text">
    <string>MexTK settings</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>txedit_CodePath_Fighter</tabstop>
  <tabstop>btn_SelectCodeFile_Fighter</tabstop>
  <tabstop>chkbox_FunctionEnable_Fighter</tabstop>
  <tabstop>txedit_FunctionPath_Fighter</tabstop>
  <tabstop>btn_SelectFunctionFile_Fighter</tabstop>
  <tabstop>chkbox_DatEnable_Fighter</tabstop>
  <tabstop>txedit_DatPath_Fighter</tabstop>
  <tabstop>btn_SelectDatFile_Fighter</tabstop>
  <tabstop>btn_Compile_Fighter</tabstop>
  <tabstop>txedit_CodePath_Item</tabstop>
  <tabstop>btn_SelectCodeFile_Item</tabstop>
  <tabstop>spinBox_ItemID</tabstop>
  <tabstop>chkbox_FunctionEnable_Item</tabstop>
  <tabstop>txedit_FunctionPath_Item</tabstop>
  <tabstop>btn_SelectFunctionFile_Item</tabstop>
  <tabstop>chkbox_DatEnable_Item</tabstop>
  <tabstop>txedit_DatPath_Item</tabstop>
  <tabstop>btn_SelectDatFile_Item</tabstop>
  <tabstop>btn_Compile_Item</tabstop>
  <tabstop>txedit_CodePath_Stage</tabstop>
  <tabstop>btn_SelectCodeFile_Stage</tabstop>
  <tabstop>chkbox_FunctionEnable_Stage</tabstop>
  <tabstop>txedit_FunctionPath_Stage</tabstop>
  <tabstop>btn_SelectFunctionFile_Stage</tabstop>
  <tabstop>chkbox_DatEnable_Stage</tabstop>
  <tabstop>txedit_DatPath_Stage</tabstop>
  <tabstop>btn_SelectDatFile_Stage</tabstop>
  <tabstop>btn_Compile_Stage</tabstop>
  <tabstop>btn_DolphinTest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
